### DISCUSSION TITLE
naticord Disscussions!

### DISCUSSION THREAD

**Discussion by n1d3v** [n1d3v/naticord#2] [General]
# Welcome to the naticord disscussions!

Here you can discuss stuff about naticord it's self, request features etc.

Have fun and be respectful.

**Comment by Shavixinio** [comment]
When will channels tab be added into Naticord?

**Reply by axeltechtips** [reply]
something like this, you think? 
![image](https://github.com/n1d3v/naticord/assets/62724735/9d1fd5f1-c45c-4df4-bebb-20cf1d416df1)

**Reply by Shavixinio** [reply]
Maybe something like this, or some kind of sidebar menu for server channels like on vanilla Discord

**Comment by axeltechtips** [comment]
I compiled and made a DMG of Naticord available to use with macOS. (friends list censored for privacy)
[download dmg](https://files.catbox.moe/a6sali.dmg)
![image](https://github.com/n1d3v/naticord/assets/62724735/f9d468b7-1fa7-4b29-831d-9f524fbacd90)
![image](https://github.com/n1d3v/naticord/assets/62724735/b2bf3695-f7bd-4b5f-94d6-6dd2d5ba2072)
![image](https://github.com/n1d3v/naticord/assets/62724735/e25818b1-be07-4eae-bc5d-0d6801addee2)

**Reply by Shavixinio** [reply]
ooomaga

**Comment by Davilarek** [comment]
> "Note that this is for Windows only but soon will be ported to Mac and Linux."

Afaik this project uses Windows Forms framework which is not portable at all. Porting to other platforms would require another almost full project rewrite.
One of alternatives to WF I could point out is AvaloniaUI.

**Reply by axeltechtips** [reply]
Naticord will have a .net version for MacOS and Linux.

**Reply by Davilarek** [reply]
Isn't it `.NET Framework` right now?

**Comment by Davilarek** [comment]
> "Discord may ban you if they catch you using a custom client, this hasn't happened to anyone yet using naticord or other client mods such as Vencord."

There is a huge difference in these terms between "Discord client **mod**" (such as Vencord) and a "Discord client". Imo this project should not compare one to another as custom clients can introduce higher risk on one's account than when using a client _mod_. Example of such risk is that the developer unintentionally could incorrectly send a really important request which may raise Discords suspicion, since it's unlikely such a mistake would happen in original client. Another example could be a bug that would spam tons of requests resulting making the account terminated due to API abuse/spam.

**Comment by tallysucks** [comment]
does anyone have the discord invite? i wanna be part of the discussion :(

**Comment by axeltechtips** [comment]
https://discord.com/invite/wMCRkZMZ
